resources/js/details.js

```diff
@@ -0,0 +1,9 @@
+function addToShoppingCart(element){
+    //Get the Text of the <p> Element of the List where the button was clicked.
+    let text = element.previousElementSibling.innerText;
```

Comment:
Should probably save IDs in the shopping card.
Would make adding links to products in the cart possible.

Reply:
You could save the IDs together with the displayed text, to make understanding the amount of an ingredient that you need, but that text would need to be grouped somehow.
Maybe something like {id: [string, string]}? Easy to check whether ingredient exists, can store multiple "descriptions" of an ingredient.

Reply:
That is very true, but that funcionality will be implemented when we know the format of the ingredients, when they are added dynamically from the backend

Reply:
```suggestion
    // TODO: save ingredient IDs instead of ingredient names. Use the IDs in the cart to group and add links to similar ingredients.
    let text = element.previousElementSibling.innerText;
```